src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -209,7 +211,7 @@ func (cfg *Config) ApplyDefaults() {
 	}
 
 	if cfg.CloudConfig == "" {
-		cfg.CloudConfig = "https://s3.amazonaws.com/lantern_config/cloud.2.0.0-nl.yaml.gz"
+		cfg.CloudConfig = fmt.Sprintf("https://s3.amazonaws.com/lantern_config/cloud.%v.yaml.gz", countryPlaceholder)
```

Comment:
@myleshorton : actually, changing this name will change the string in the lantern.yaml.  If we ever want to change the `cloudconfig` for old clients again we'll need to remember we need to set `cloudconfig` **and** `cloudconfigurl` to the new value so we're sure to hit old clients.  Not a huge deal, but I don't think the name is either.  I'd rather punt on this kind of stuff until we have working auto-updates and thus we don't need to care much about backwards compatibility anymore.  But if you still want it, it's a quick change.